.travis.yml

```diff
@@ -0,0 +1,4 @@
+language: node_js
+node_js:
+  - "6"
+  - "4"
```

Comment:
good point sir, was more of a copy paste of a different module. Does `latest` work as option? Checking..